pyproject.toml

```diff
@@ -32,7 +32,7 @@ rich = { version = "^2.2.5", optional = true }
 async_exit_stack = "^1.0.1"
 jsonrpc-base = "^1.0.3"
 
-ya-aioclient = "0.3.0"
+ya-aioclient = "^0.4.0"
```

Comment:
@mfranciszkiewicz didn't it require an update?

Reply:
no, it will download the latest 0.4.x version